From d9e8b21ddd0222c306a92efa56a7bad5429d8c17 Mon Sep 17 00:00:00 2001
From: Louis Abel <label@rockylinux.org>
Date: Tue, 8 Aug 2023 12:54:03 -0700
Subject: [PATCH] Ensure cloud-user is applied to OpenELA

---
 config/cloud.cfg.tmpl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/cloud.cfg.tmpl b/config/cloud.cfg.tmpl
index 08b6efb..ed82d8d 100644
--- a/config/cloud.cfg.tmpl
+++ b/config/cloud.cfg.tmpl
@@ -2,7 +2,7 @@
 # The top level settings are used as module
 # and system configuration.
 {% set is_bsd = variant in ["dragonfly", "freebsd", "netbsd", "openbsd"] %}
-{% set is_rhel = variant in ["rhel", "centos"] %}
+{% set is_rhel = variant in ["rhel", "centos", "openela"] %}
 {% if is_bsd %}
 syslog_fix_perms: root:wheel
 {% elif variant in ["suse"] %}
-- 
2.41.0

